frontend/workflows/k8s/src/resize-hpa.tsx

```diff
@@ -40,6 +41,20 @@ const HPADetails: React.FC<WizardChild> = () => {
     hpaData.updateData(key, value);
   };
 
+  const metadata = [];
+
+  if (hpa.annotations) {
+    Object.keys(hpa.annotations).forEach(key => {
```

Comment:
nit: recommend lodash forEach

Reply:
@danielhochman , jst want to triple check this is what you had in mind https://github.com/lyft/clutch/pull/794/commits/6c86a0a9f48c588c9d4a20a05a790fc292603443.

could have also done `_.keys(something).forEach(key => {`

Reply:
yep looks good!